lib/conduit_lwt_tls.mli

```diff
@@ -32,9 +32,10 @@ module Server : sig
     Tls.Config.server ->
     Lwt_unix.file_descr ->
     (Lwt_unix.file_descr * Lwt_io.input_channel * Lwt_io.output_channel) Lwt.t
+  [@@@deprecated "Low level function. Will be removed."]
```

Comment:
If they are removed `init` must provide a way to pass a `server`. I think the functions in this module overall should leave more choice to the user (i.e. do not force "no authentication" to the client…). Since users can directly access this module it does not have to have the same interface as `conduit_lwt_unix`.